packages/cli/src/lib/commands/collect/options/index.ts

```diff
@@ -1,29 +1,31 @@
-import { CollectYargsOptions, PersistYargsOptions } from './types';
-import { AssertYargsOptions } from '../../assert/options/types';
-import { param as openReport } from './openReport';
-import { param as ufPath } from './ufPath';
-import { param as configPath } from './configPath';
-import { param as outPath } from './outPath';
-import { param as url } from './url';
-import { param as format } from './format';
-import { param as serveCommand } from './serveCommand';
-import { param as awaitServeStdout } from './awaitServeStdout';
-import { param as dryRun } from './dryRun';
-import { ASSERT_OPTIONS } from '../../assert/options';
+import { InferredOptionTypes, Options } from 'yargs';
+import { openReport } from './openReport';
+import { ufPath } from './ufPath';
+import { configPath } from './configPath';
+import { outPath } from './outPath';
+import { url } from './url';
+import { format } from './format';
+import { serveCommand } from './serveCommand';
+import { awaitServeStdout } from './awaitServeStdout';
+import { dryRun } from './dryRun';
+import { assertOptions } from '../../assert/options';
+import { config } from './config';
 
-export const PERSIST_OPTIONS: PersistYargsOptions = {
-  ...outPath,
-  ...format,
-  ...openReport
-};
+export const persistOptions = {
+  outPath,
+  format,
+  openReport
+} satisfies Record<string, Options>;
```

Comment:
It is enough to check if the type satisfies the Options type. This fixes the error when combining options and we do not need to build a type or use and type casting.